index.bs

```diff
@@ -1410,6 +1433,18 @@ Its value is 50.
 maximum [=set/size=] of an [=attribution source=]'s [=attribution scopes/values=].
 Its value is 20.
 
+<dfn>Max length per aggregatable bucket for source</dfn> is a positive integer that controls
+the maximum [=string/length=] of an [=attribution source=]'s
```

Comment:
Maybe "the maximum length of keys of an attribution source's aggregatable attribution bucket budget and ..."?